### PR TITLE
Scene Class Fixed

### PR DESCRIPTION
**Pull request by iBreak97**
Simple Fix Of Scene Class

### CODE REVIEW

File: src/Netsphere.Resource/Scene/Chunks/ModelChunk.cs [hunk 1]
```diff
@@ -262,10 +265,15 @@ public virtual void Serialize(Stream stream)
                 {
                     w.WriteCString(texture.FileName, 1024);
                     if (Unk1 >= 0.2000000029802322f)
-                        w.WriteCString(texture.FileName2, 1024);
+                    {
+                      byte[] fileName2 = Encoding.ASCII.GetBytes(texture.FileName2);
+                      w.Write(fileName2, 0, 1024);
+                      //w.WriteCString(texture.FileName2, 1024);
+                    }
```
Comment (wtfblub) [inline src/Netsphere.Resource/Scene/Chunks/ModelChunk.cs]: This doesn't change anything. WriteCString does the same thing.

File: src/Netsphere.Resource/Scene/Chunks/ModelChunk.cs [hunk 2]
```diff
@@ -289,7 +297,11 @@ public virtual void Deserialize(Stream stream)
                     };
 
                     if (Unk1 >= 0.2000000029802322f)
-                        textureData.FileName2 = r.ReadCString(1024);
+                    {
+                      char[] fileName2 = r.ReadChars(1024);
+                      textureData.FileName2 = new string(fileName2);
+                      //textureData.FileName2 = r.ReadCString(1024);
+                    }
```
Comment (wtfblub) [inline src/Netsphere.Resource/Scene/Chunks/ModelChunk.cs]: This doesn't change anything. WriteCString does the same thing.

File: src/Netsphere.Resource/Scene/SceneContainer.cs [hunk 3]
```diff
@@ -183,7 +183,7 @@ public void Serialize(Stream stream)
         {
             using (var w = stream.ToBinaryWriter(true))
             {
-                //w.Write(Version);
+                w.Write(Version);
```
Comment (wtfblub) [inline src/Netsphere.Resource/Scene/SceneContainer.cs]: Is this actually needed? I removed Version because it caused some issues in the past for me.

**Comment by iBreak97** [comment]
WriteCString not work...

**Comment by wtfblub** [comment]
08939b4895e72b479a5ee64d74e5c61311556120

**Pull request closed without merging**